hadoop-ozone/common/src/test/java/org/apache/hadoop/ozone/om/helpers/TestOmKeyInfo.java

```diff
@@ -145,7 +145,7 @@ private void createdAndTest(boolean isMPU) {
 
     // Because for OmKeyLocationInfoGroup we have not implemented equals()
     // method, so it checks only references.
-    Assert.assertNotEquals(key, cloneKey);
+    Assert.assertEquals(key, cloneKey);
```

Comment:
Reason behind this line change? Was this test broken?

Reply:
Oh, got it.

Now that we implemented equals() in OmKeyLocationInfoGroup, should we change the command right above this line as well?

Reply:
I've updated the comment accordingly.

Reply:
I was out for a while . Thanks @smengcl for updating the PR :)